src/config/config.server.json

```diff
@@ -0,0 +1,4 @@
+{
+    "issuerServerUrl": "http://localhost:5108",
```

Comment:
Do we need to change this? I get

<img width="587" alt="Screen Shot 2021-08-17 at 01 01 14" src="https://user-images.githubusercontent.com/36084092/129661726-ed4e389a-659d-4046-aa21-a18adf68da57.png">

Reply:
Yes, this is part of https://github.com/rsksmart/email-vc-issuer/pull/49 that needs to be deployed first, or around the same time.

Reply:
I have updated the code to point to the correct location. I also moved all of the credential code into the /src/features/credentials.ts file. It did not belong in the state folder.